### ISSUE TITLE
Cozy shouldnt create folders in ~ by default

### ISSUE THREAD

**Issue by Tids** [geigi/cozy#308]
### Bug/Feature description
Cozy shouldn't create 2 additional folders inside of our home. This is very inflexible (like, some of us export `xdg-user-dir MUSIC` and `xdg-user-dir VIDEO` folders for a good reason to another drive) and it just looks bad to have 2 new folders in there without any icon or real belonging.
A better place would be to have them in `xdg-user-dir MUSIC` by default so it will respect our xdg-user-dir setup.
Basically this: https://gitlab.gnome.org/GNOME/gnome-sound-recorder/-/issues/20
but with different starting and end-points.

**Comment by geigi** [comment]
Thanks for reporting :) 
I'll have to do some research how this will interact with popular music players. I assume most people wouldn't want their audiobooks also in their music player. If people use the music directory in the home folder this would be the case. The user would need to add an exception in the settings of the music player to exclude the audiobook folder.

Right now you can change the location of your audiobooks in the settings and you can also specify multiple locations for your audiobooks. After changing the location you can delete the audiobooks folder created by cozy.
